### ISSUE TITLE
future / plans

### ISSUE THREAD

**Issue by fopsdev** [redpelicans/aurelia-material#11]
are there any plans to update this project? (loved the demo and the blog so far)

**Comment by eric-basley** [comment]
we had a look on Aurelia when Google decided to stop Angular1, but yet we definitely migrated to React ecosystem and have no time to update this project. So I can try to be more reactive to pull requests but no more.

**Comment by DarienFord** [comment]
@eric-basley my team is all in on Aurelia and mdlite.  Would you be open to transferring ownership of the repo to us?  Alternatively we can use my fork as the starting point.

We will be updating to the latest mdl-lite.

**Comment by eric-basley** [comment]
Yes I'm completely open. Could you please tell me more about your team and
you ?
Regards
Eric

Eric BASLEY
+33 6 85 80 63 16
www.redpelicans.com
